src/styles/blocks/clock__min.scss

```diff
@@ -0,0 +1,19 @@
+.clock {
```

Comment:
in case you have a BEM block .clock you don't need to create more files. 
Just create one scss file clock.scss and use min and sec hands inside

```
.clock {
  &__min {
    ...
  }
  
  &__sec {
    ...
  }
}
```